### DISCUSSION TITLE
Fabric.js 6 How should I implement an eraser

### DISCUSSION THREAD

**Discussion by StrawberryCake-Fish** [fabricjs/fabric.js#10086] [Q&A] [answered]
Hi everyone, I am a beginner and I am learning Fabric.js. I need to implement an artboard, but I don't know how to implement an eraser. I want my strokes and eraser marks to be integrated so that when I move, I don't just remove one. Can someone help me? Thank you very much.

**Comment by asturur** [comment, marked as answer]
there is a package you could look at:
https://github.com/ShaMan123/erase2d
You can ask more there
